lib/test-collection.js

```diff
@@ -64,7 +66,15 @@ TestCollection.prototype.add = function (test) {
 			throw new Error('"only" cannot be used with a ' + type + ' test');
 		}
 
-		this.hooks[type].push(test);
+		// add a always hook
+		if (metadata.always) {
+			if (type !== 'after' && type !== 'afterEach') {
+				throw new Error('"always" cannot be used with a ' + type + ' test');
```

Comment:
@cgcgbcbc - Can you add a few tests in `test/test-collection.js` for this throwing behavior?